.travis.yml

```diff
@@ -1,15 +1,20 @@
 language: python
+
 python:
   - "3.6"
 
 cache: pip
 
+env:
+  matrix:
+    - TOX_ENV=python
+    - TOX_ENV=flake8
+
 install:
-  - "pip install -r requirements.txt"
-  - pip install coveralls
+  - pip install --upgrade pip
+  - pip install tox coveralls
 
 script:
-  - coverage run setup.py test
+  - tox -e $TOX_ENV
 
-after_success:
-  coveralls
+after_success: test $TOX_ENV = "python" && coveralls
```

Comment:
We need to verify that we only push to coveralls after we run the "python" test environment.